public/javascripts/store.js

```diff
@@ -140,6 +140,35 @@ class Store {
     }
   }
 
+   // Update the PaymentIntent with the the currency and payment value.
+   async updatePaymentIntentWithCurrencyPaymentMethod(
+    paymentIntent,
+    currency,
+    payment_methods,
+  ) {
```

Comment:
Looks like the formatting got squiffy here.

Reply:
Fixed the formatting.